src/node.cc

```diff
@@ -2849,66 +2853,127 @@ void ProcessEmitWarning(Environment* env, const char* fmt, ...) {
   f.As<v8::Function>()->Call(process, 1, &arg);
 }
 
+static bool PullFromCache(Environment* env,
+                          const FunctionCallbackInfo<Value>& args,
+                          Local<String> module,
+                          Local<Object> cache) {
+  Local<Object> exports;
+  Local<Context> context = env->context();
+  if (cache->Has(context, module).FromJust()) {
+    exports = cache->Get(context, module).ToLocalChecked()->
+        ToObject(context).ToLocalChecked();
```

Comment:
You could replace the Has/Get with a single Get, like this:
```c++
Local<Value> exports_v;
Local<Object> exports;
if (cache->Get(context, module).ToLocal(&exports_v) &&
    exports_v->IsObject() &&
    exports_v->ToObject(context).ToLocal(&exports)) {
  args.GetReturnValue().Set(exports);
  return true;
}
return false;
```

Reply:
TIL, moved to your approach